zig/download.sh

```diff
@@ -76,17 +76,22 @@ else
 fi
 
 # Extract and then remove the downloaded tarball:
-echo "Extracting $ZIG_TARBALL..."
 tar -xf "$ZIG_TARBALL"
 rm "$ZIG_TARBALL"
 
-# Replace any existing Zig installation so that we can install or upgrade:
-echo "Installing $ZIG_DIRECTORY to 'zig' in current working directory..."
-rm -rf zig
-mv "$ZIG_DIRECTORY" zig
+# Replace these existing directories and files so that we can install or upgrade:
+rm -rf zig/doc
+rm -rf zig/lib
+mv "$ZIG_DIRECTORY/LICENSE" zig
+mv "$ZIG_DIRECTORY/README.md" zig
+mv "$ZIG_DIRECTORY/doc" zig
+mv "$ZIG_DIRECTORY/lib" zig
+mv "$ZIG_DIRECTORY/zig" zig
```

Comment:
A nit, and the old script didn't do this, but using a trailing slash (`zig/`) to indicate the target is a directory is nice.